sympy/ntheory/primetest.py

```diff
@@ -110,6 +110,31 @@ def mr(n, bases):
     return True
 
 
+def isprimeE(n, b):
```

Comment:
isprime_euler or is_euler_pseudoprime would be a better name.

Reply:
Yeah it sounds more relating
I changed it
Thanks for the suggestion.